### DISCUSSION TITLE
Using Postal 2.0

### DISCUSSION THREAD

**Discussion by biswajitnet** [postalserver/postal#1669] [Help with using Postal]
How can we  export database dump for receiving Hardfail/bounce information. 
What tables should we be exporting ?

**Comment by willpower232** [comment]
what are you trying to do?

**Reply by biswajitnet** [reply]
Thanks @willpower232  for responding, I am trying to export a dump of all hardfails/bounces till date so I can mark them in my EMA.

**Reply by willpower232** [reply]
the official method would be to make use of webhooks so that Postal can update your software with whether messages are delivered or not
